druid/src/ext_event.rs

```diff
@@ -117,6 +118,25 @@ impl ExtEventSink {
         ));
         Ok(())
     }
+
+    /// Schedule an idle callback.
```

Comment:
Could you add a few more details here? Like:
- the data must match the root data
- it will get run on the main thread

Maybe it's also worth copying or linking other details from the `druid_shell::IdleHandle` docs? I'd lean towards copying instead of linking, because `IdleHandle` seems to be mostly intended for druid itself, and not for druid users.